lib/web/users.go

```diff
@@ -169,7 +169,13 @@ func updateUser(r *http.Request, m userAPIGetter) (*ui.User, error) {
 
 	user.SetRoles(req.Roles)
 
-	updateUserTraits(req, user)
+	// we do not want situation where one trait set (allTraits)
+	// overrides another (traits).
```

Comment:
I think the comment is slightly confusing, I would rephrase to indicate that in some cases clients of this API just send builtin traits to set and some send the entire traits set to set.